assets/src/dashboard/app/views/templateDetails/content/detailsGallery/index.js

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useMemo } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { sprintf, __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+  CardGallery,
+  ColorList,
+  PaginationButton,
+  Pill,
+} from '../../../../../components';
+import { TemplatePropType } from '../../../../../types';
+import {
+  ByLine,
+  Column,
+  ColumnContainer,
+  DetailContainer,
+  LargeDisplayPagination,
+  MetadataContainer,
+  SmallDisplayPagination,
+  Text,
+  Title,
+} from './../../components';
+
+function DetailsGallery({
+  activeTemplateIndex,
+  isRTL,
+  orderedTemplatesLength,
+  switchToTemplateByOffset,
+  template,
+}) {
+  const { NextButton, PrevButton } = useMemo(() => {
+    const Previous = (
+      <PaginationButton
+        rotateRight={true}
```

Comment:
We can just remove the `true` here

Reply:
Yeah not bad to add in. It's really easy to leave a `true` in if it was previously dynamic or you were testing `false`.